library/jruby

```diff
@@ -1,5 +1,5 @@
-Maintainers: Brian Goff <cpuguy83@gmail.com> (@cpuguy83)
-GitRepo: https://github.com/cpuguy83/docker-jruby.git
+Maintainers: JRuby Admin <admin@jruby.org> (@jruby)
+GitRepo: https://github.com/jruby/docker-jruby.git
```

Comment:
This being a new repo instead of a moved repo means we lose issues/PRs, which isn't great -- is there a particular reason moving @cpuguy83's existing repo didn't work out?

Reply:
Only that I did not want to move it before this transfer is official... chicken and egg. If it is fine to move it directly I am sure @cpuguy83 can help us make that change. The JRuby repo can be dumped at any time.

Reply:
It's looking pretty official to me :eyes: :wink: :heart:

Reply:
Ok, I will work with @cpuguy83 to transfer the repo directly to @jruby organization. Nothing will visibly change related to this PR.

Reply:
Transfer complete. https://github.com/jruby/jruby/issues/5598#issuecomment-749180697

Reply:
Can confirm!  Clicking https://github.com/cpuguy83/docker-jruby/blob/510284ff5ffbb257937d755a9d351e9df50ba31b/9000/jre8/Dockerfile (from https://hub.docker.com/_/jruby) takes me to https://github.com/jruby/docker-jruby/blob/510284ff5ffbb257937d755a9d351e9df50ba31b/9000/jre8/Dockerfile properly! :tada: :heart: